tests/foreman/cli/test_organization.py

```diff
@@ -903,24 +988,24 @@ def test_positive_add_capsule_by_name(self):
             'name': org['name'],
             'smart-proxy': proxy['name'],
         })
+        org = Org.info({'name': org['name']})
+        self.assertIn(proxy['name'], org['smart-proxies'])
 
     @run_only_on('sat')
     @tier2
     @stubbed()
     def test_positive_add_capsule_by_id(self):
-        """@test: Add a capsule by its ID
+        """Add a capsule to organization by its ID
 
         @feature: Organization
 
         @assert: Capsule is added to the org
-
-        @status: manual
         """
 
     @run_only_on('sat')
     @stubbed("Needs to be re-worked!")
```

Comment:
So we are using `stubbed` to note that the tests that are still yet to be implemented.  Also we automatically skip all `stubbed` tests. See https://github.com/SatelliteQE/robottelo/blob/master/Makefile#L13. But this test seem to have some code and it needs some update so it is technically partially implemented already.